docs/testplan/GCU-Dynamic-ACL-testplan.md

```diff
@@ -267,9 +307,103 @@ DYNAMIC_ACL_TABLE | DYNAMIC_ACL_TABLE_TYPE | {vlan port 1} | DYNAMIC_ACL_TABLE_T
 - Operation Success
 
 **Additional Checks**
-+ Check that results of “show acl rule DYNAMIC_ACL_TABLE RULE_1” and “show acl rule DYNAMIC_ACL_TABLE RULE_2” match the following output:
-  + DYNAMIC_ACL |_TABLE	RULE_1 | 9999 | FORWARD |  DST_IP: 103.23.2.1/32
-  + DYNAMIC_ACL_TABLE | RULE_2 | 9998 | FORWARD | DST_IPV6: 103.23.2.1::1/128
++ Check that results of “show acl rule DYNAMIC_ACL_TABLE RULE_1” and “show acl rule DYNAMIC_ACL_TABLE RULE_2” match the following output rows respectively:
+  + Table             | Rule   |Priority| Action| Match | Status
+    ----------------- | -------|------| -------| ------------------------| ----
+    DYNAMIC_ACL_TABLE |	RULE_1 | 9999 | FORWARD |  DST_IP: 103.23.2.1/32 | Active
+  + Table | Rule | Priority | Action | Match | Status
+    ----- | ---- | ---- | ---- | ---- | -----
+    DYNAMIC_ACL_TABLE | RULE_2 | 9998 | FORWARD | DST_IPV6: 103.23.2.1::1/128 | Active
+
+### Create DHCP Forwarding Rules
+
+**Json Patch:**
+
+    [
+        {
+            "op": "add",
+            "path": "/ACL_RULE",
+            "value": {
+                "DYNAMIC_ACL_TABLE|DHCP_RULE": {
+                    "IP_PROTOCOL": "17",
+                    "L4_DST_PORT": "67",
+                    "ETHER_TYPE": "0x0800",
+                    "PRIORITY": "9999",
+                    "PACKET_ACTION": "FORWARD"
+                }
+            }
+        }
+    ]
+
+**Expected Result**
+- Operation Success
+
+**Additional Checks**
++ Check that results of “show acl rule DYNAMIC_ACL_TABLE DHCP_RULE” match the following output:
+  + Table | Rule | Priority | Action | Match | Status
+    ---- | ---- | ------- | ------- | ------| ------
+    DYNAMIC_ACL_TABLE |	DHCP_RULE | 9999 | FORWARD |  L4_DST_PORT: 67 | Active
+    | | | | | ETHER_TYPE: 0x0800 |
+    | | | | | IP_PROTOCOL: 17 |
+
+### Create DHCPv6 Forwarding Rules
+
+**Json Patch:**
+
+    [
+        {
+            "op": "add",
+            "path": "/ACL_RULE",
+            "value": {
+                "DYNAMIC_ACL_TABLE|DHCPV6_RULE": {
+                    "IP_PROTOCOL": "17",
+                    "L4_DST_PORT_RANGE": "547-548",
+                    "ETHER_TYPE": "0x86DD",
+                    "PRIORITY": "9998",
+                    "PACKET_ACTION": "FORWARD"
+                }
+            }
+        }
+    ]
+
+**Expected Result**
+- Operation Success
+
+**Additional Checks**
++ Check that results of “show acl rule DYNAMIC_ACL_TABLE DHCPV6_RULE” match the following output:
+  + Table | Rule | Priority | Action | Match | Status
+     ---- | ---- | ------- | ------- | ------| ------
+     DYNAMIC_ACL_TABLE |	DHCPV6_RULE | 9998 | FORWARD |  L4_DST_PORT_RANGE: 547-548 | Active
```

Comment:
Why check port 548 for DHCPv6? Seems it's for AFP protocal?

Reply:
Ah, typo on my part, meant for that to be 546-547 for DHCPv6 Client and Server ports, will update

Reply:
Pushed up with fixed port range @yaqiangz

Reply:
@jongorel In my understanding of DHCPv6, seems you want to verify both server side (port 547) and client side (port 546). But in DHCPv4, you only mentioned server side port 67 (It's used for client to relay or relay communicate with server). Is this inconsistence by design?

Reply:
@yaqiangz this is by design.  During manual testing of applying these acl's then running the existing DHCP_Relay/DHCPv6_Relay tests, DHCP_Relay passed with only server side port traffic forwarded, but DHCPv6_Relay seemed to be dropping an advertise packet and was failing every time until I allowed forwarding to Client side port as well